examples/README.md

```diff
@@ -0,0 +1,26 @@
+# Usage
+`npm install` -> Installs all required dependencies to run these examples.
+
+## Different Examples
+**Basic Usage**: Create asset and transfer it to new owner. 
+-> `npm start`
+
+**Async/Await Basic Usage**: Basic usage example rewritten with async/await.
+-> `npm run basic-async`
+
+**Querying for Assets**: Query for assetdata or metadata.
+-> `npm run query-assets`
+
+**Seed/Keypair Functionality**: Create keypair with bip39 library.
+-> `npm run seed-func`
+
+## Notes
```

Comment:
I would rather include this Notes part at the beginning, as otherwise the  Different Examples do not work.